src/main/java/com/mcmoddev/lib/init/Fluids.java

```diff
@@ -92,15 +90,16 @@ protected static BlockFluidClassic addFluidBlock(@Nonnull final MMDMaterial mate
 			return null;
 		}
 
-		if (!name.equals(MaterialNames.MERCURY)) {
-			block = new BlockFluidClassic(material.getFluid(), Material.LAVA);
-		} else {
+		// TODO: make it into a flag/stat or something
+//		if (!name.equals(MaterialNames.MERCURY)) {
+//			block = new BlockFluidClassic(material.getFluid(), Material.LAVA);
```

Comment:
line 95 is the generic bit, the rest which you've inadvertently force enabled is mercury specific